modules/services/getmail.nix

```diff
@@ -0,0 +1,193 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.getmail;
+
+  retrieverModule = types.submodule {
+    options = {
+      type = mkOption {
+        type = types.enum [
+          "SimplePOP3Retriever"
```

Comment:
This submodule should not be necessary;  the config is already available in modules/accounts/email.nix: you get the username, the passwrod, the ssl support for imap and pop from there.

Reply:
Many thanks for the pointer(s). Will adjust and update.